### PR TITLE
Feature: Updated the design for startup settings

### PR DESCRIPTION
**Pull request by heftymouse**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [x] Did you remove any strings from the en-us resource file?
   - [x] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots (optional)**
image

### CODE REVIEW

File: src/Files.App/Views/Settings/GeneralPage.xaml [hunk 1]
```diff
@@ -120,66 +120,73 @@
...
+												VerticalAlignment="Center"
 												Text="{x:Bind Text}"
 												TextTrimming="CharacterEllipsis"
 												ToolTipService.ToolTip="{x:Bind Text}" />
-										</DataTemplate>
-									</ListView.ItemTemplate>
-								</ListView>
-							</Grid>
-						</local:SettingsBlockControl>
+											<Button
```
Comment (yaira2) [inline src/Files.App/Views/Settings/GeneralPage.xaml:179]: Can you add automation properties?

File: src/Files.App/Views/Settings/GeneralPage.xaml [hunk 2]
```diff
@@ -120,66 +120,73 @@
...
+											Text="{helpers:ResourceString Name=Home}" />
+										<MenuFlyoutItem Command="{x:Bind ViewModel.AddPageCommand}" Text="{helpers:ResourceString Name=Browse}" />
+									</MenuFlyout>
+								</Button.Flyout>
+								<StackPanel Orientation="Horizontal" Spacing="8">
+									<FontIcon
+										Margin="0,1,0,0"
+										FontSize="16"
+										Glyph="&#xE710;" />
+									<TextBlock Text="Add Page" />
```
Comment (yaira2) [inline src/Files.App/Views/Settings/GeneralPage.xaml:146]: Should this be localized?

File: src/Files.App/Views/Settings/GeneralPage.xaml [hunk 3]
```diff
@@ -120,66 +120,73 @@
...
+										<MenuFlyoutItem
+											Command="{x:Bind ViewModel.AddPageCommand}"
+											CommandParameter="Home"
+											Text="{helpers:ResourceString Name=Home}" />
+										<MenuFlyoutItem Command="{x:Bind ViewModel.AddPageCommand}" Text="{helpers:ResourceString Name=Browse}" />
+									</MenuFlyout>
+								</Button.Flyout>
+								<StackPanel Orientation="Horizontal" Spacing="8">
+									<FontIcon
+										Margin="0,1,0,0"
```
Comment (yaira2) [inline src/Files.App/Views/Settings/GeneralPage.xaml:143]: Is this margin needed? It's a bit of an odd number..

File: src/Files.App/Views/Settings/GeneralPage.xaml.cs [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
 {
 	public sealed partial class GeneralPage : Page
 	{
+
 		public GeneralPage()
 		{
 			InitializeComponent();
 		}
+
+		private void DeleteTag(object sender, RoutedEventArgs e)
```
Comment (yaira2) [inline src/Files.App/Views/Settings/GeneralPage.xaml.cs:18]: Is this a copy and paste? It should probably be `RemoveStartupPage`.

Comment (heftymouse) [reply src/Files.App/Views/Settings/GeneralPage.xaml.cs:18]: Whoops, it's not a copy paste but I happened to be looking at the tabs page right before adding that

File: src/Files.App/Views/Settings/GeneralPage.xaml (hunk 3 above)
Comment (heftymouse) [reply src/Files.App/Views/Settings/GeneralPage.xaml:143]: The + icon looks off centre without it

File: src/Files.App/Views/Settings/GeneralPage.xaml.cs (hunk 4 above)
Comment (yaira2) [reply src/Files.App/Views/Settings/GeneralPage.xaml.cs:18]: I've done this before 🙂

File: src/Files.App/Views/Settings/GeneralPage.xaml (hunk 3 above)
Comment (yaira2) [reply src/Files.App/Views/Settings/GeneralPage.xaml:143]: Does a vertical alignment help?

Comment (heftymouse) [reply src/Files.App/Views/Settings/GeneralPage.xaml:143]: Unfortunately not, the extra space seems to be part of the glyph itself

Comment (yaira2) [reply src/Files.App/Views/Settings/GeneralPage.xaml:143]: The issue with using `1` is that it'll only work in some scales.

Comment (Jay-o-Way) [reply src/Files.App/Views/Settings/GeneralPage.xaml:143]: Yeah, it's tempting to "manually" tweak margins like this, but generally: just don't. Especially not with odd numbers like this.

File: src/Files.App/Views/Settings/GeneralPage.xaml [hunk 5]
```diff
@@ -140,10 +140,10 @@
 								</Button.Flyout>
 								<StackPanel Orientation="Horizontal" Spacing="8">
 									<FontIcon
-										Margin="0,1,0,0"
-										FontSize="16"
```
Comment (Jay-o-Way) [inline src/Files.App/Views/Settings/GeneralPage.xaml]: General recommendation is **always** use sizes in a multiple of 4. Please use 16.

File: src/Files.App/Views/Settings/GeneralPage.xaml [hunk 6]
```diff
@@ -140,10 +140,10 @@
...
+										VerticalAlignment="Center"
+										FontSize="14"
```
Comment (Jay-o-Way) [inline src/Files.App/Views/Settings/GeneralPage.xaml:144]: General recommendation is **always** use sizes in a multiple of 4. Please use 16.

Comment (yaira2) [reply src/Files.App/Views/Settings/GeneralPage.xaml:144]: @Jay-o-Way Thank you for the suggestion. The original reason for this rule doesn't apply anymore (it was originally for Windows Phone) but we still follow it in most cases as it's a good way to have a consistent design.

Comment (Jay-o-Way) [reply src/Files.App/Views/Settings/GeneralPage.xaml:144]: I think it's still valid. I've seen it explicitly mentioned in Win 11 documentation. Reason is scaling factors, which also apply to desktops. Icons are also designed with a base size of 16 pixels. Using 14 will cause some blurring effects.

File: src/Files.App/Utils/RecentItem/RecentItems.cs [hunk 7]
```diff
@@ -114,7 +113,6 @@ public async Task<List<RecentItem>> ListRecentFilesAsync()
 		/// </summary>
 		public async Task<List<RecentItem>> ListRecentFoldersAsync()
 		{
-			var recentItems = new List<RecentItem>();
```
Comment (yaira2) [inline src/Files.App/Utils/RecentItem/RecentItems.cs:117]: Is this related, or should it be reverted?

Comment (heftymouse) [reply src/Files.App/Utils/RecentItem/RecentItems.cs:117]: The variable was previously unused, so the change might as well stay

File: src/Files.App/Views/Settings/GeneralPage.xaml.cs [hunk 8]
```diff
@@ -1,15 +1,23 @@
...
 {
 	public sealed partial class GeneralPage : Page
 	{
+
 		public GeneralPage()
 		{
 			InitializeComponent();
 		}
+
+		private void DeleteStartupPage(object sender, RoutedEventArgs e)
```
[the remark this answers is not in the document]
Comment (heftymouse) [reply src/Files.App/Views/Settings/GeneralPage.xaml.cs]: Changed

**Review by yaira2 — APPROVED** [review]
LGTM
